platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/config/DefaultConfiguration.java

```diff
@@ -87,11 +86,9 @@ public static Configuration buildBasicConfiguration(
             @NonNull final Path settingsPath, @NonNull final List<Path> configurationPaths) throws IOException {
         final ConfigSource settingsConfigSource = LegacyFileConfigSource.ofSettingsFile(settingsPath);
         final ConfigSource mappedSettingsConfigSource = ConfigMappings.addConfigMapping(settingsConfigSource);
-        final ConfigSource threadCountPropertyConfigSource = new ThreadCountPropertyConfigSource();
 
-        final ConfigurationBuilder configurationBuilder = ConfigurationBuilder.create()
-                .withSource(mappedSettingsConfigSource)
-                .withSource(threadCountPropertyConfigSource);
```

Comment:
Not needed anymore? I remember there was a config that default value depends on the cpu count.

Reply:
Only needed by FCHashMap, which is now all static configuration (and no longer production code).